src/ctia/entity/incident.clj

```diff
@@ -143,6 +143,13 @@
     (assoc-in [:intervals interval]
               (jt/time-between (jt/instant earlier) (jt/instant later) :seconds))))
 
+(def new-status-set #{"New" "New: Processing" "New: Presented"})
+
+(def open-status-set #{"Open" "Open: Investigating" "Open: Reported" "Open: Contained" "Open: Recovered"})
+
+(def closed-status-set #{"Closed" "Closed: Under Review" "Closed: Confirmed Threat" "Closed: Suspected"
+                         "Closed: False Positive" "Closed: Near-Miss" "Closed: Other" "Closed: Merged"})
```

Comment:
That's the most rational to have sets, but that's likely that we will need to add other variations.
So perhaps we could work with regexes:
``` clojure
(defn new-status? [status]
   (some? (re-matches #"New(: .+)?" status)))
  
(defn closed-status? [status]
   (some? (re-matches #"Closed(: .+)?" status)))
```

I let you choose :-)